integration-tests/gatsby-source-wordpress/__tests__/index.js

```diff
@@ -52,6 +58,23 @@ describe(`[gatsby-source-wordpress] Run tests on develop build`, () => {
       await gatsbyCleanBeforeAll()
     }
 
+    if (isWarmCache && !process.env.WORDPRESS_BASIC_AUTH) {
+      console.log(
```

Comment:
What do you think about doing something like this instead?

```suggestion
      throw new Error(/* ...*/)
      
```

Reply:
If I remember right it doesn't stop the tests from running when an error is thrown here. Which seems odd to me but this was the only way I could get it to properly exit

Reply:
I was going for "nice visible warning" but it sounds like this takes care of that as is :+1: